samples/testing/operations/test_operations.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import qsharp
+from qsharp.utils import dump_operation
+
+@pytest.fixture(autouse=True)
+def setup():
+    """Fixture to execute asserts before a test is run"""
+    # You can set the target profile here if you want
+    # By default, it uses "Unrestricted" profile
+    qsharp.init(target_profile=qsharp.TargetProfile.Base)
+    yield # this is where the testing happens
```

Comment:
Several questions on this piece:
* The doc comment mentions asserts but the code doesn't actually include asserts
* I understand that this executes qsharp.init before every test, correct? But all tests except the first one call qsharp.init anyways, re-initializing qsharp again and not benefitting from this fixture?
* You shouldn't need to specify profile at all, the tests can run without that, and here you want to have the minimal test code that achieves the goal.
* I would swap the init argument for `project_root='.'`, since you do actually need that for the later tests and they don't work without it. Right now, the later tests call init three times, one from setup, one with setting target profile, and one with setting project root, and I think only the last one is actually used